src/components/flat-table/flat-table.stories.mdx

```diff
@@ -1732,6 +1731,222 @@ of the button control.
+          <FlatTableCell>Single</FlatTableCell>
+          <FlatTableCell>
+            <ActionPopover>
+              <ActionPopoverItem
+                onClick={() => {}}
+                submenu={
+                  <ActionPopoverMenu>
+                    <ActionPopoverItem onClick={() => {}}>
+                      CSV
+                    </ActionPopoverItem>
+                    <ActionPopoverItem onClick={() => {}}>
+                      PDF
+                    </ActionPopoverItem>
+                  </ActionPopoverMenu>
+                }
+              >
+                action
+              </ActionPopoverItem>
+              <ActionPopoverItem
+                onClick={() => {}}
+                submenu={
+                  <ActionPopoverMenu>
+                    <ActionPopoverItem onClick={() => {}}>
+                      CSV
+                    </ActionPopoverItem>
+                    <ActionPopoverItem onClick={() => {}}>
+                      PDF
+                    </ActionPopoverItem>
+                  </ActionPopoverMenu>
+                }
+              >
+                action
+              </ActionPopoverItem>
+            </ActionPopover>
+          </FlatTableCell>
+        </FlatTableRow>,
+      ];
+      const [recordsRange, setRecordsRange] = useState({ start: 0, end: 5 });
+      const [currentPage, setCurrentPage] = useState(1);
+      const renderRows = () => {
+        const { start, end } = recordsRange;
+        if (start < 0) return rows;
+        if (end > rows.length) return rows.slice(start, rows.length);
+        return rows.slice(start, end);
+      };
+      const handlePagination = (newPage, newPageSize) => {
+        const start = (newPage - 1) * newPageSize;
+        const end = start + newPageSize;
+        setPlacementUp(newPageSize !== 1);
+        setRecordsRange({ start, end });
+        setCurrentPage(newPage);
+      };
+      const getMinHeight = (numberOfRows) => {
+        if (!numberOfRows || numberOfRows > rows.length - 1) return minHeight;
+        return refs.slice(0, numberOfRows).reduce((acc, ref) => {
+          acc += ref.current?.getBoundingClientRect().height;
+          return acc;
+        }, 80)
+      }
+      const [minHeight, setMinHeight] = useState()
+      const [rowCount, setRowCount] = useState(3);
+      useLayoutEffect(() => {
+        setMinHeight(getMinHeight(rowCount));
+      }, [])
+      useEffect(() => {
+        setMinHeight(getMinHeight(rowCount));
+      }, [rowCount])
+      return (
+        <div style={{ height: "220px", marginBottom: "100px" }}>
+          <Button mb={1} mr={1} disabled={rowCount <= 1} onClick={() => setRowCount((prev) => prev - 1)}>Decrease min rows</Button>
+          <Button mb={1} disabled={rowCount >= rows.length - 1} onClick={() => setRowCount((prev) => prev + 1)}>Increase min rows</Button>
+          <FlatTable
+            minHeight={minHeight}
+            height="100px"
```

Comment:
I removed the `height="100px"` because we shouldn't need it as well here, but it broke the whole example so I think something is going wrong somewhere

Before you look into it, do we even need to provide this example at all? All we've done is add a `minHeight` prop, the implementation is then down to the user. IMO this is overkill and will just muddy up the docs a bit

Reply:
Yeah for the `minHeight` to be demoed you either need to set the `height` on the table or on the wrapper. I take your point about it being overkill for storybook, I'll just use this in the sandbox to show the linked issue has been fixed

Reply:
Actually looking at again I think it's related to it defaulting the height to `100%` if the sticky props are used and no height is passed